packages/website/docs/core/inputs_transform/TextSplitter.md

```diff
@@ -0,0 +1,83 @@
+## TextSplitter
+
+The `TextSplitter` is a text splitting callable event designed to split a given text into multiple contexts based on specified parameters. It extends the functionality of ContextSplitter by allowing the text to be split using a custom separator and controlling whether the separator should be retained in the outputs.
+
+---
+
+| Reference     | Link                                                                                                                                                                         |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Encre Concept | [Callable Event](**-a-link-to-the-corresponding-concept-documentation-**)                                                                                                    |
+| Encre Node    | [TextSplitterNode](**-a-link-to-the-corresponding-node-documentation-**)                                                                                                     |
+| API           | [TextSplitter](**-a-link-to-the-corresponding-api-documentation-**)                                                                                                          |
+
+### Overview
+
+The `TextSplitter` is a text splitting callable event designed to split a given text into multiple contexts based on specified parameters. It extends the functionality of ContextSplitter by allowing the text to be split using a custom separator and controlling whether the separator should be retained in the outputs.
```

Comment:
This is same as the brief description of the TextSplitter. You might want to change this or the brief description.